pkg/universe.dagger.io/alpha/kubernetes/helm/upgrade.cue

```diff
@@ -81,10 +87,10 @@ import (
 
 	// used to avoid name clashes
 	let releaseName = name
+	let environment = env
```

Comment:
You could also just put `"` around `env` in `docker.#Run` (see my related comment below)

Reply:
I will change this today. I wanted to add quotes around some shell variables as well.